### ISSUE TITLE
test_vcs_basic_meshfill_-3_proj_gmflip_NH_via_gm.png is not correct

### ISSUE THREAD

**Issue by danlipsa** [UV-CDAT/uvcdat#1905]
Image is:
![image](https://cloud.githubusercontent.com/assets/6180727/14216273/ef5596fc-f815-11e5-9aed-393da8ca25fe.png)

Test command is:

"/home/danlipsa/build/uvcdat/install/bin/python" "/home/danlipsa/src/uvcdat/testing/vcs/test_vcs_basic_gms.py" "--gm_type=meshfill" "--projection=-3" "--lat1=90" "--lat2=0" "--range_via_gm" "--gm_flips_lat_range" "--source=/home/danlipsa/build/uvcdat/uvcdat-testdata/baselines/vcs/test_vcs_basic_meshfill_-3_proj_gmflip_NH_via_gm.png"

(lat1, lat2) = (0, 90) after the filip, so the view should be a doughtnut viewed from the South Pole

**Comment by doutriaux1** [comment]
looks like the continents are cropped correctly but not the data

**Comment by danlipsa** [comment]
That is right. I think this is related to the fact that vcs.utils.getworldcoordinates does not work correctly for meshfil.
